tools/peptidegenomiccoordinate/peptideGenomicCoodinate.xml

```diff
@@ -0,0 +1,58 @@
+<tool id="peptidegenomiccoodinate" name="Peptide Genomic Coodinate" version="0.1.1">
+    <description>Get Peptide's genomic coordinate using mzsqlite DB and genomic mapping sqlite DB</description>
+    <requirements>
+        <requirement type="package" version="2.7.9">python</requirement>
+    </requirements>
+    <command detect_errors="aggressive"><![CDATA[
+        python '$__tool_directory__/peptideGenomicCoodinate.py' 
+        '$peptideinput'
+        '$mzsqlite'
+        '$mapping'
+        '$peptide_bed'
+    ]]></command>
+    <inputs>
+        <param type="data" name="peptideinput" format="tabular" label="Peptide List (without any header line)"/>
+        <param type="data" name="mzsqlite" format="sqlite" label="mz to sqlite (mzsqlite) file"/>
+        <param type="data" name="mapping" format="sqlite" label="genomic mapping sqlite file"/>
+    </inputs>
+    <outputs>
+        <data format="bed" name="peptide_bed" label="${tool.name} on ${on_string}">
+            <actions>
+                <action name="column_names" type="metadata" default="chrom,chromStart,chromStop,name,score,strand,thickStart,thickEnd,itemRgb,blockCount,blockSizes,blockStarts"/>
+            </actions>
+        </data>
+    </outputs>
+    <tests>
+        <test>
+            <param name="peptideinput" value="peptides.tabular"/>
+            <param name="mzsqlite" value="test_mz_to_sqlite.sqlite"/>
```

Comment:
This file is too large for this test ... its nearly 20MB. We should keep the test data as small as possible ...